File: ViewModels/DataTableRequestVM.cs
using System.Collections.Generic;

namespace ViewModels
{
    ///Start - JSon class sent from Datatables
    
    public class DataTableRequestVM
    {
        public int Draw { get; set; }
        public int Start { get; set; }
        public int Length { get; set; }
        public List<ColumnVM> Columns { get; set; }
        public SearchVM Search { get; set; }
        public List<OrderVM> Order { get; set; }
    }

    public class ColumnVM
    {
        public string Data { get; set; }
        public string Name { get; set; }
        public bool Searchable { get; set; }
        public bool Orderable { get; set; }
        public SearchVM Search { get; set; }
    }

    public class SearchVM
    {
        public string Value { get; set; }
        public string Regex { get; set; }
    }

    public class OrderVM
    {
        public int Column { get; set; }
        public string Dir { get; set; }
    }

    /// End- JSon class sent from Datatables 
}

File: ViewModelsVB/DataTableRequestVM.vb
REM Start - JSon class sent from Datatables

Public Class DataTableRequestVM
    Public Property Draw As Integer
    Public Property Start As Integer
    Public Property Length As Integer
    Public Property Columns As List(Of ColumnVM)
    Public Property Search As SearchVM
    Public Property Order As List(Of OrderVM)
End Class

Public Class ColumnVM
    Public Property Data As String
    Public Property Name As String
    Public Property Searchable As Boolean
    Public Property Orderable As Boolean
    Public Property Search As SearchVM
End Class

Public Class SearchVM
    Public Property Value As String
    Public Property Regex As String
End Class


Public Class OrderVM
    Public Property Column As Integer
    Public Property Dir As String
End Class

REM End- JSon class sent from Datatables